workflow_scheduler_test.go

```diff
@@ -203,12 +221,39 @@ func TestSubscriptionCallback_QueryWithCondition_2(t *testing.T) {
 	time.Sleep(5 * time.Second)
 }
 
+func TestSubscriptionCallback_OldEvent(t *testing.T) {
+	// Case 4: Call the SubscriptionCallback method with an event older than MaxAgeToProcessWinEvtsInDays in the config.
+	// Requires MaxAgeToProcessWinEvtsInDays in the test config is 1.
+
+	// Assume
+	ctrl := gomock.NewController(t)
```

Comment:
Explain how if the mockWorkflows functions get called, the Controller will error, failing this test